### PR TITLE
Update Styles.xml

### PR DESCRIPTION
**Pull request by AliAyub007**
Removing Title bar and giving app a professional and better user interactive interface.

Fixes issue #86 
Changes: Removing Title bar and giving app a professional and better user interactive interface.

Screenshots for the change:

### CODE REVIEW

**Comment by dynamitechetan** [comment]
Action bar is an important design element that provides a consistent familiar look between Android apps. It also provides a space for the app or activity’s identity, thus enabling the user to know their location in the app, and easy access to the actions that can be performed.
I think it should be there in the app.

**Comment by AliAyub007** [comment]
I was thinking it in a way that there is a navigation bar in app which is showing position to user so that's why I thought of it.

**Comment by dynamitechetan** [comment]
But there are other activities like Crop, Edit Image etc. which will require action bar. So why not to have a consistent look for the entire app.

**Comment by AliAyub007** [comment]
yeah that is a valid point. (Y)

File: app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -2,9 +2,12 @@
 
     <!-- Base application theme. -->
     <style name="AppTheme" parent="Phimpme.Base">
+        <item name="colorPrimary">@color/colorPrimary</item>
+        <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
+        <item name="colorAccent">@color/colorAccent</item>
     </style>
 
-    <style name="Phimpme.Base" parent="Theme.AppCompat.Light.DarkActionBar">
+    <style name="Phimpme.Base" parent="Theme.AppCompat.Light.NoActionBar">
```
Comment (Abhi2424shek) [inline app/src/main/res/values/styles.xml]: We want an action bar for sure.

**Comment by Abhi2424shek** [comment]
I am closing this PR as the change is not needed! I hope you understand, next time comment on a relevant issue before making a PR.

**Pull request closed without merging**